From c9fa13942d3082ab122c8f3ac58d0a71198cbe4d Mon Sep 17 00:00:00 2001
From: Matthias Clasen <mclasen@redhat.com>
Date: Wed, 2 Nov 2011 22:52:17 -0400
Subject: [PATCH] Don't call g_thread_init()

g_type_init() has been doing this anyway since GLib 2.24 or so
anyway. And with GLib 2.31, it is not necessary anymore.
---
 src/yelp.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/src/yelp.c b/src/yelp.c
index 6cf425b..fac6d1a 100644
--- a/src/yelp.c
+++ b/src/yelp.c
@@ -37,7 +37,6 @@ main (int argc, char **argv)
 {
     YelpApplication *app;
 
-    g_thread_init (NULL);
     g_type_init ();
 
     setlocale (LC_ALL, "");
-- 
1.7.7

